toolkit/tools/internal/exe/exe.go

```diff
@@ -40,6 +40,11 @@ func OutputDirFlag(k *kingpin.Application, doc string) *string {
 	return k.Flag("output-dir", doc).Required().String()
 }
 
+// LogColorFlag registers a log color flag for k and returns the passed value
+func LogColorFlag(k *kingpin.Application) *string {
```

Comment:
Since we are adding this to every tool, do we want to consider doing something like `SetupProfileFlags()` which just handles logging as a single entity? We need to do 3x calls in every tool now. I'm fine leaving the separate flags if we want,  but maybe we want a single convenience function that generates the struct, and update `logger.InitBestEffort()` to just take that struct as a parameter? Or add a new version.

Reply:
A second function would also allow the changes to all the tools to be bundled into a separate PR.